### PR TITLE
STENCIL-3126: Add pull request template for contribution guidelines

### PR DESCRIPTION
**Pull request by bc-AlyssNoland**
See title. CONTRIBUTING.md has a link to this which currently 404s.

### CODE REVIEW

**Comment by bc-AlyssNoland** [comment]
@bigcommerce/stencil-team Any input welcome

**Comment by junedkazi** [comment]
I would rather add a github template for new pull request based on https://help.github.com/articles/creating-an-issue-template-for-your-repository & just point the url to "New Pull Request" link  (https://github.com/bigcommerce/stencil/pull/new/master)

**Comment by pedelman** [comment]
Might consider moving files to `.github` directory so we dont clutter the root directory of the project.

**Comment by bc-AlyssNoland** [comment]
@pedelman done

@junedkazi so someone more slimmed down and change the PR link in CONTRIBUTING.md?

**Comment by junedkazi** [comment]
I did not mean moving CONTRIBUTING.md under `.github`. I meant more like updating the broken link to  https://github.com/bigcommerce/stencil/pull/new/master

**Comment by bc-AlyssNoland** [comment]
@junedkazi Okay. I can do that. I was also making the change that Patrick requested.

**Comment by junedkazi** [comment]
yup I had requested the same change with a link on how to do it :)

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### Expected behavior
```
Comment (junedkazi) [inline .github/ISSUE_TEMPLATE.md]: Can we file this as a separate PR ? Also we need to talk internally if we want to open up the issue queue.

**Comment by bc-AlyssNoland** [comment]
@mcampa I don't believe I do, but should anything go into the changelog for this?

**Comment by junedkazi** [comment]
No changelog is required for this change.

**Pull request merged**